### DISCUSSION TITLE
Help with generating a reaction of my fixture

### DISCUSSION THREAD

**Discussion by Hooli486** [EnviralDesign/GeoPix#33] [Q&A]
Hello there!

So, I just started with GeoPix, watched quite a bit tutorials and I understand what is happening (i think). However, I just started my own project and I tried to replicate the simple fixture in one of the tutorials of 32x32. For some reason, when checking if all the pixels react, not all pixels react, only the standard 10x10 grid reacts. 

![afbeelding](https://user-images.githubusercontent.com/87446344/125944663-9e36d2de-d49a-43cd-9006-8f190771e925.png)

![afbeelding](https://user-images.githubusercontent.com/87446344/125944716-63f9929a-ade6-4010-9553-331575ca56b6.png)

I followed the exact steps of the tutorial, and I tried to figure out myself what could be going wrong. For some reason, the generator is only recognising the original 10x10 grid and not the new 32x32 grid. It already goes wrong with the generator, because even without the projector and checking if the whole 32x32 grid works, by clicking on the fixture in the outliner field, the grid does not fully work, but only the 10x10 grid. 

I do get the following errors:
![afbeelding](https://user-images.githubusercontent.com/87446344/125945141-47b2c3c3-2ba6-475b-b597-09f9300fc34d.png)

I tried looking for a solution myself (also just started with a new file a couple of times and tried other things with the fixture), however I'm too much of a noob and I keep having the same problem. And I hope somebody else instead can help me figure out this probably quite easy problem :D

**Comment by EnviralDesign** [comment]
@Hooli486 thanks for writing about this.

It turns out there's a bug that you're running into that I just ran into myself doing some tests last night. I am looking into a fix today, and should have something soon. 

Thanks for the report!

**Reply by Hooli486** [reply]
Thanks for the quick reaction, sounds good, looking forward to trying out the programme after the bug fix :D

**Reply by EnviralDesign** [reply]
alright bug's been fixed ! Thanks again.

**Reply by Hooli486** [reply]
No problem!
